custom/icds_reports/reports/incentive.py

```diff
@@ -18,12 +18,12 @@ def get_excel_data(self):
         excel_rows = []
         excel_rows.append(headers)
         for row in data:
-            home_visit_percent = row['valid_visits'] / row['expected_visits']
-            weighing_efficiency = row['wer_weighed'] / row['wer_eligible']
+            home_visit_percent = row['valid_visits'] / row['expected_visits'] if row['expected_visits'] != 0 else 1
```

Comment:
is this equivalent to:

`row['valid_visits'] / (row['expected_visits'] if row['expected_visits'] != 0 else 1)`

or

`(row['valid_visits'] / row['expected_visits']) if row['expected_visits'] != 0 else 1`

If its the first one, then, if `valid_visits > 1` then this value will be > 100%. If the second one, it would always be 100%. which one is expected?

Similar question for weighing efficiency in the next line

Reply:
The second one. The indicator should show as 100% if the denominator is zero. (needed a way to not penalize workers if they had no scheduled visits, it effectively says they missed 0 visits in that case)